modules/bullet/godot_result_callbacks.cpp

```diff
@@ -47,8 +47,45 @@ bool godotContactAddedCallback(btManifoldPoint &cp, const btCollisionObjectWrapp
 	return true;
 }
 
+uint32_t get_mask(uint32_t mask) {
+	return mask & 0x007fffff;
+}
+
+uint32_t get_team_bits(uint32_t mask) {
+	return mask & 0xff000000;
+}
+
+bool is_ally_only_mode(uint32_t mask) {
+	return (mask & 0x00800000) > 0;
```

Comment:
I think I'd appreciate a breakdown of the mask values in the comments using them, as it's not immediately obvious why we're using these magic numbers. AKA that bits 1 through to 23 are open range but 24 through to 32 are being reserved to encode data to be used in the collision filter test.